src/python/grpcio_health_checking/grpc_health/v1/health.py

```diff
@@ -55,18 +61,20 @@ def next(self):
     def __next__(self):
         return self._next()
 
-    def add(self, response):
+    def add(self, response: Any) -> None:
```

Comment:
Can we supply a better type than `Any`? I believe this should be `HealthCheckResponse`

Reply:
updated types